spring-integration-debezium/src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+	<Appenders>
+		<Console name="Console" target="SYSTEM_OUT">
+			<PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n" />
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="org.junit" level="warn" />
+		<Logger name="example" level="warn" />
+		<Logger name="org.testcontainers" level="INFO"/>
+		<Logger name="com.github.dockerjava" level="WARN"/>
+		<Logger name="org.springframework.integration.debezium" level="info"/>
+
+		<Logger name="org.junit.vintage.engine.discovery" level="error" />
+
+		<Root level="error">
+			<AppenderRef ref="Console" />
+		</Root>
+	</Loggers>
+</Configuration>
```

Comment:
Please, ensure that every file comes with a new line in the end.
Would be great if your IDE could do that on save.